Explorer/Assets/Scripts/CrdtEcsBridge/ComponentWriter/ECSToCRDTWriter.cs

```diff
@@ -0,0 +1,70 @@
+using CRDT;
+using CRDT.Protocol;
+using CRDT.Protocol.Factory;
+using CrdtEcsBridge.OutgoingMessages;
+using CrdtEcsBridge.Serialization;
+using System;
+using System.Collections.Generic;
+
+namespace CrdtEcsBridge.ECSToCRDTWriter
+{
+    public class ECSToCRDTWriter : IECSToCRDTWriter
```

Comment:
You had an example system that used the writer, I propose to design a generic barebone that processes components to see how it will be used in the future.

Reply:
and add a test with an implementation of a such generic system that will prove the direction